layers/+completion/helm/funcs.el

```diff
@@ -622,3 +622,11 @@ to buffers)."
   (interactive)
   (let (helm-candidate-number-limit)
     (helm-themes)))
+
+;; Buffers ---------------------------------------------------------------------
+
+(defun spacemcacs/helm-buffers-list-unfiltered ()
```

Comment:
Function name prefix typo: `spacemcacs`
same with the function call for the `bU` key binding on line 85.

Reply:
Done ! ✨